src/EPR.RegulatorService.Frontend.Web/Controllers/RegistrationSubmissions/RegistrationSubmissionsController_part.cs

```diff
@@ -54,17 +54,6 @@ private bool GetOrRejectProvidedSubmissionId(Guid? submissionId, out Registratio
             return true;
         }
 
-        private bool GetAndRememberSubmissionDetails(Guid? submissionId, out RegistrationSubmissionDetailsViewModel model)
```

Comment:
It would be nice to see this method handle this logic more appropriately, clean the controller up

Reply:
I don't think this method implemented correctly. not required out param, it's  not support async calls.

Reply:
I like clean the controller up... but this code were defines with in control only.